src/framework/audio/CMakeLists.txt

```diff
@@ -277,23 +282,22 @@ endif()
 
 if (MUSE_MODULE_AUDIO_JACK)
     find_package(Jack REQUIRED)
-    set(MODULE_INCLUDE_PRIVATE ${MODULE_INCLUDE_PRIVATE} ${JACK_INCLUDE_DIRS} )
-    set(MODULE_LINK ${MODULE_LINK} ${JACK_LIBRARIES} pthread )
+    set(MODULE_INCLUDE_PRIVATE ${MODULE_INCLUDE_PRIVATE} ${JACK_INCLUDE_DIRS})
+    set(MODULE_LINK ${MODULE_LINK} ${JACK_LDFLAGS} pthread)
+endif()
+
+if (OS_IS_MAC)
+    find_library(AudioToolbox NAMES AudioToolbox)
+    find_library(CoreAudio NAMES CoreAudio)
+    set(MODULE_LINK ${MODULE_LINK} ${AudioToolbox} ${CoreAudio})
+elseif (OS_IS_WIN AND (NOT (MINGW)))
+    set(MODULE_LINK ${MODULE_LINK} winmm mmdevapi mfplat)
+elseif (OS_IS_LIN)
     find_package(ALSA REQUIRED)
-    set(MODULE_INCLUDE_PRIVATE ${MODULE_INCLUDE_PRIVATE} ${ALSA_INCLUDE_DIRS} )
-    set(MODULE_LINK ${MODULE_LINK} ${ALSA_LIBRARIES} pthread )
-else ()
-    if (OS_IS_MAC)
-        find_library(AudioToolbox NAMES AudioToolbox)
-        find_library(CoreAudio NAMES CoreAudio)
-        set(MODULE_LINK ${MODULE_LINK} ${AudioToolbox} ${CoreAudio})
-    elseif (OS_IS_WIN)
-        set(MODULE_LINK ${MODULE_LINK} winmm mmdevapi mfplat)
-    elseif (OS_IS_LIN)
-        find_package(ALSA REQUIRED)
-        set(MODULE_INCLUDE_PRIVATE ${MODULE_INCLUDE_PRIVATE} ${ALSA_INCLUDE_DIRS} )
-        set(MODULE_LINK ${MODULE_LINK} ${ALSA_LIBRARIES} pthread )
-    endif()
+    set(MODULE_INCLUDE_PRIVATE ${MODULE_INCLUDE_PRIVATE} ${ALSA_INCLUDE_DIRS})
+    set(MODULE_LINK ${MODULE_LINK} ${ALSA_LIBRARIES} pthread)
+elseif (MINGW)
+    set(MODULE_LINK ${MODULE_LINK} winmm mmdevapi mfplat)
```

Comment:
The `elseif (OS_IS_WIN AND (NOT (MINGW)))` and `elseif (MINGW)` cases can be combined